features/bevragen/persoon/verblijfplaats/dev/protocollering-vraag-gba.feature

```diff
@@ -0,0 +1,66 @@
+#language: nl
+
+Functionaliteit: protocollering van de vraag voor verblijfplaats
+
+  Rule: Met fields gevraagde velden worden geprotocolleerd als de elementnummers volgens Logisch ontwerp BRP
+    Dit is een 6-cijferige code, met zo nodig voorloopnul voor categorieën.
+    Ook alle velden die worden meegeleverd voor het bepalen van het verblijfplaats type worden vastgelegd in het veld 'request_zoek_rubrieken'.
+
+    Abstract Scenario: Met fields vragen om <fields> wordt vastgelegd als zoekrubrieken <zoek rubrieken>
+      Gegeven de persoon met burgerservicenummer '000000012' heeft de volgende gegevens
+      | naam            | waarde     |
+      | anummer (01.10) | 8468619643 |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                               | waarde   |
+      | gemeente van inschrijving (09.10)  | 0518     |
+      | functie adres (10.10)              | W        |
+      | datum aanvang adreshouding (10.30) | 20190714 |
+      | datum ingang geldigheid (85.10)    | 20190714 |
+      En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+      | naam                                       | waarde           |
+      | gemeente_code                              | 0518             |
+      | straatnaam (11.10)                         | Borgesiusstraat  |
+      | naam openbare ruimte (11.15)               | Borgesiusstraat  |
+      | huisnummer (11.20)                         | 103              |
+      | huisletter (11.30)                         | b                |
+      | huisnummertoevoeging (11.40)               | 2                |
+      | postcode (11.60)                           | 2497BV           |
+      | woonplaats (11.70)                         | Scheveningen     |
+      | identificatiecode verblijfplaats (11.80)   | 0518010000747448 |
+      | identificatiecode nummeraanduiding (11.90) | 0518200000747446 |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000012                       |
+      | fields              | <fields>                        |
+      Dan is protocollering opgeslagen met de volgende gegevens
```

Comment:
Is het een idee om de dan stap als volgt te wijzigen:

      Dan heeft de persoon met anummer '1234567890' de volgende 'protocollering' gegevens
      | anummer    | request_zoek_rubrieken |
      | 1234567890 | <zoek rubrieken>       |

Voordeel hiervan is dezelfde stijl als 'Dan heeft de response een persoon met de volgende gegevens'.
Verder kan de automation code zo worden geïmplementeerd dat alleen de kolommen wordt gevalideerd die zijn opgegeven. Voordeel hiervan is dat bij antwoord scenario's de request_zoek_rubrieken kolom niet hoeft te worden opgegeven. Dit geldt ook voor de anummer kolom

Reply:
aangepast zoals voorgesteld